src/Filter.js

```diff
@@ -32,6 +32,7 @@ export default class Filter extends Component {
   }
 
   changeArea = event => {
+    debugger 
```

Comment:
Just remember not to include debuggers when you push up. :)

Reply:
Thanks for catching that! I will remove that immediately